pkg/network/ovn_kubernetes_test.go

```diff
@@ -84,8 +85,9 @@ func TestRenderOVNKubernetes(t *testing.T) {
 			},
 		},
 	}
+	fakeClient := cnofake.NewFakeClient() // TODO add a number of nodes
 
-	objs, _, err := renderOVNKubernetes(config, bootstrapResult, manifestDirOvn)
+	objs, _, err := renderOVNKubernetes(config, bootstrapResult, manifestDirOvn, fakeClient)
```

Comment:
in the future should we add new test cases here to test the logic of what phase we are in and what happens in failure scenarios? maybe you already added these somewhere and I'm missing them.

Reply:
that's missing now :-/
Do you think this can this be added later?